codefresh.yml

```diff
@@ -0,0 +1,26 @@
+version: '1.0'
+stages:
+  - clone
+  - test  
+steps:
+  main_clone:
+    title: Clone repository
+    stage: clone
+    type: git-clone
+    repo: ${{CF_REPO_OWNER}}/${{CF_REPO_NAME}}
+    revision: ${{CF_SHORT_REVISION}}
+    git: github
+  setup_github_token:
+    title: Setup GitHub token
+    stage: clone
+    image: codefresh/cli
+    commands:
+      - cf_export GITHUB_TOKEN=$(codefresh get context github --decrypt -o yaml | yq -y .spec.data.auth.password)
+  tests:
+    stage: test        
+    title: Run code quality
+    image: gcr.io/linio-support/docker-php:7.4
+    commands:      
+      - test -n "${GITHUB_TOKEN}" && composer config -g github-oauth.github.com ${GITHUB_TOKEN}
```

Comment:
We don't need the test part. That's done in the `Dockerfile` because it's not needed for every project and we didn't want to have to add that in there manually if someone does set it.